src/router_core/router_core_private.h

```diff
@@ -321,7 +321,8 @@ ALLOC_DECLARE(qdr_router_ref_t);
 DEQ_DECLARE(qdr_router_ref_t, qdr_router_ref_list_t);
 
 typedef enum {
-    QDR_DELIVERY_NOWHERE = 0,
+    QDR_DELIVERY_UNINITIALIZED = 0,
+    QDR_DELIVERY_NOWHERE,
```

Comment:
I claim that QDR_DELIVERY_NOWHERE is the proper default value for this type.  This type is used to indicate in which list a particular delivery is contained.  If it has never been inserted into a list, it should indicate NOWHERE.